src/Treeview.ts

```diff
@@ -277,13 +277,15 @@ export class JsonOutlineProvider implements vscode.TreeDataProvider<string> {
                     if (keyNode.properties[i].name._value.toUpperCase() === "type".toUpperCase()) {
                         let resourceType = keyNode.properties[i].value._value.toUpperCase();
                         icon = "resources.svg";
+                        // tslint:disable:no-unused-expression
```

Comment:
Urrgg there are so many cleaner ways to do this but I'll let you use the "Not my code" excuse if you want haha

Reply:
Yep. :-)

Reply:
Mostly didn't want to have to create templates with all those resource types to test.  Added grandfather clause.